appcat/objectbucket/composition.yaml

```
@@ -0,0 +1,91 @@
+---
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: default-objectbucket-cloudscale
```

```suggestion
  name: objectbucket-cloudscale
```

Not sure if the "default" in the name brings much added value...

---

With the latest clarifications about the XRD no, it does not add any value.